index.html

```diff
@@ -40,6 +40,128 @@ <h1 class="my-name">I’m Mariane <br> Glad to see <br> you!</h1>
+                    <p class="work-desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="categories">
+                        <li>Html</li>
+                        <li>Css</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button class="btn-primary">See Project</button>
+                </div>
+            </div>
+
+            <div class="work-card">
+                <div class="work-img">
+                    <img src="img/gym.png" width="275" alt="">
+                </div>
+                <div class="work-details">
+                    <h2>Workout</h2>
+                    <div class="experience">
+                        <h4 class="title">Gym</h4>
+                        <p class="role"> <img src="/img/Counter.png" alt=""> Back End Dev</p>
+                        <p class="year"> <img src="/img/Counter.png" alt=""> 2016</p>
+                    </div>
+                    <p class="work-desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="categories">
+                        <li>Html</li>
+                        <li>Css</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button class="btn-primary">See Project</button>
+                </div>
+            </div>
+            <div class="work-card">
+                <div class="work-img">
+                    <img src="img/printing.png" width="275" alt="">
+                </div>
+                <div class="work-details">
+                    <h2>Professional Printing Works</h2>
+                    <div class="experience">
+                        <h4 class="title">Print</h4>
+                        <p class="role"> <img src="/img/Counter.png" alt=""> Back End Dev</p>
+                        <p class="year"> <img src="/img/Counter.png" alt=""> 2016</p>
+                    </div>
+                    <p class="work-desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="categories">
+                        <li>Html</li>
+                        <li>Css</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button class="btn-primary">See Project</button>
+                </div>
+            </div>
+        </section>
+
+        <section id="about">
+            <div class="about-content">
+                <h2>About <br>myself</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <div>
+                    <ul class="social-media-icon">
+                        <li><img src="/img/social icons/twiter icon.svg" alt=""></li>
```

Comment:
yes, thanks. But what about those ones that do not have spaces in them? They are not showing at your end. I just and everything seems to be working well here. I have one review left. please I suggest it will be best for me if you can check again. Thanks.

Reply:
I have also updated the media names as you suggested. Thanks